src/Compilers/Core/MSBuildTask/ValidateBootstrap.cs

```diff
@@ -77,10 +79,38 @@ public override bool Execute()
             // named pipe errors, CPU load causing timeouts, etc ...  Hence flagging a single failure would produce
             // a lot of false positives.  The current value was chosen as a reasonable number for warranting an 
             // investigation.
-            if (s_failedServerConnectionCount > 20)
+            const int maxRejectCount = 5;
+            var rejectCount = 0;
+            foreach (var tuple in s_failedQueue.ToList())
             {
-                Log.LogError($"Too many compiler server connection failures detected: {s_failedServerConnectionCount}");
-                allGood = false;
+                switch (tuple.ResponseType)
+                {
+                    case ResponseType.AnalyzerInconsistency:
+                        Log.LogError($"Analyzer inconsistency building {tuple.OutputAssembly}");
```

Comment:
How do we know which analyzers had inconsistencies when this happens? Are other diagnostic messages printed in this scenario?

Reply:
No. There are no other diagnostic.

At the moment this isn't a user visible diagnostic. This is deliberately an implementation detail because this can happen in completely well functioning build. Surfacing a real diagnostic here would essentially be giving users false and unactionable information.

In this very specific scenario, a controlled bootstrap build, we know that this is wrong and hence we can emit an error.